executor/src/witgen/processor.rs

```diff
@@ -98,7 +99,7 @@ pub struct Processor<'a, 'b, 'c, T: FieldElement, Q: QueryCallback<T>> {
     /// The values of the publics
     publics: BTreeMap<&'a str, T>,
     /// The mutable state
-    mutable_state: &'c mut MutableState<'a, 'b, T, Q>,
+    mutable_state: &'c MutableState<'a, 'b, T, Q>,
```

Comment:
A few lines above, there is a comment explaining the 3 lifetimes, including:
```rs
/// - `'b`: The duration of this machine's call (e.g. the mutable references of the other machines)
```

I think this is no longer accurate?

Reply:
I think it's the lifetime of the query callback.